docs/adr/0003-graphql-layer-between-backend-frontend.md

```diff
@@ -0,0 +1,98 @@
+# GraphQL layer between backend and frontend
+
+Date: 2020-04-30
+
+## Status
+
+**Accepted**
```

Comment:
Not accepted yet

Reply:
True. In light of recent discussions, this has not yet been accepted.